contracts/starknet/lib/eip712.cairo

```diff
@@ -69,19 +87,15 @@ namespace EIP712 {
         alloc_locals;
 
         let voter_address = calldata[0];
-
         let (authenticator_address) = get_contract_address();
         let (auth_address_u256) = FeltUtils.felt_to_uint256(authenticator_address);
 
         // Ensure voter has not already used this salt in a previous action
         let (already_used) = EIP712_salts.read(voter_address, salt);
-
         with_attr error_message("EIP712: Salt already used") {
             assert already_used = 0;
         }
 
-        // We don't need to pad because calling `.address` with starknet.js
-        // already left pads the address with 0s
```

Comment:
Might be interesting to keep this comment?

Reply:
See my comment in the issue you posted :) I think we should still keep the comment! :)